lib/membrane_http_adaptive_stream/manifest/track.ex

```diff
@@ -185,25 +229,40 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest.Track do
   def supports_partial_segments?(%__MODULE__{target_partial_segment_duration: duration}),
     do: duration != nil
 
-  @doc """
-  Add a segment of given duration to the track.
-  It is recommended not to pass discontinuity attribute manually but use `discontinue/1` function instead.
-  """
```

Comment:
why remove all the docs and comments?

Reply:
all of them will be placed back, I removed them because ElixirLS was underlying it all the time and it was irritating during development.